### DISCUSSION TITLE
Upstream compilers?

### DISCUSSION THREAD

**Discussion by azukaitis** [spack/spack#24727] [General]
I'm having a bit of trouble trying to get compilers from an upstream... is this supposed to work?

Tony

**Comment by scheibelp** [comment]
To make sure: I'm guessing you want to install compilers in an upstream Spack instance, and then use those compilers to build packages in a local Spack instance. In that case, you will still need to `spack compiler add` those compilers (e.g. ``spack compiler find `spack location -i gcc`/bin``) or set `install_missing_compilers: True` in the config (in the latter case, Spack may still reinstall a compiler depending on the circumstances).
